src/App.jsx

```diff
@@ -14,33 +16,93 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button type="button" className="button is-info is-light">
-        Sort alphabetically
-      </button>
+const SORT_FIELD = {
+  alphabeticaly: 'alphabetic',
```

Comment:
There is a typo in the key 'alphabeticaly'. It should be 'alphabetically'. This could cause issues when trying to sort alphabetically.